### PR TITLE
Random screaming when hurt

### PR DESCRIPTION
**Pull request by eeSPee**
## About The Pull Request

Now there is a chance your (human) character will scream when taking over 10 brute/burn. The more damage, the higher the chance. Cannot occur when the character is dead/incapacitated.

## Why It's Good For The Game

Muh immersion.

## Changelog
:cl:
tweak: people scream when taking damage
/:cl:

### CODE REVIEW

**Comment by zeskorion** [comment]
Bruh rng bad

**Comment by PowerfulBacon** [comment]
Use if(prob(chance)) instead of that thing you did

**Comment by AnjelicaChaykovsky** [comment]
Oh i saw it on other servers very cool

**Comment by Isy232** [comment]
This is cool.

File: code/modules/mob/living/carbon/human/damage_procs.dm [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 /mob/living/carbon/human/apply_damage(damage = 0,damagetype = BRUTE, def_zone = null, blocked = FALSE, forced = FALSE)
 	// depending on the species, it will run the corresponding apply_damage code there
+	if (stat != DEAD && (damagetype==BRUTE || damagetype==BURN) && damage>10 && prob(10+damage/2))	
+		say("*scream")
```
Comment (PowerfulBacon) [inline code/modules/mob/living/carbon/human/damage_procs.dm]:
```suggestion
		emote("scream")
```

**Review by qwertyquerty — APPROVED** [review]

**Comment by qwertyquerty** [comment]
@ike709 broken checks

**Pull request merged**